src/engine/renderer/gl_shader.cpp

```diff
@@ -1041,11 +1039,6 @@ void GLShaderManager::CompileGPUShaders( GLShader *shader, shaderProgram_t *prog
 			/* FIXME: add this test: ( strcmp( token, "USE_TCGEN_LIGHTMAP" ) == 0 && r_lightMapping->integer == 0 )
 			when lightmaps are never used when lightmapping is disabled
 			see https://github.com/DaemonEngine/Daemon/issues/296 is fixed */
-			if ( strcmp( token, "USE_LIGHT_MAPPING" ) == 0 && r_vertexLighting->integer != 0 )
```

Comment:
The comment preceding this line is now meaningless (also `is fixed` was a leftover).

Maybe rewrite the comment saying we may want to not build it when surface isn't lightmapped or lightmapping is disabled but this currently does not work (with a link to the issue/PR)?

Reply:
I don't understand it well enough to write an intelligent comment myself.